app/src/main/java/org/eyeseetea/malariacare/DashboardActivity.java

```diff
@@ -763,6 +764,19 @@ public void onTabChanged(String tabId) {
         }
     }
 
+    /**
+     * Handles resolution callbacks.
+     */
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode,
```

Comment:
@xurxodev  It is called here: https://github.com/EyeSeeTea/pictureapp/pull/1375/files#diff-93401c54e92d24bdf56298493f2c7799R93